packages/lib/modules/vebal/cross-chain/CrossChainBoost.tsx

```diff
@@ -42,7 +43,7 @@ export function CrossChainBoost() {
 
   const [syncIsOpen, setSyncIsOpen] = useState(false)
 
-  const { myVebalBalance } = useVebalUserData()
+  const { veBALBalance } = useVebalUserData()
```

Comment:
We are calling `formatUnits(veBALBalance, 18)` in many places. Why not exposing a new `veBalBalanceUnits` from `useVebalUserData`?

Reply:
Will probably first wait till we change all the calculations to use the raw bigint value, and then check if it is used just for formatting. Probably a formatting function without side effects?